rebar.config

```diff
@@ -16,6 +16,10 @@
        , {docsh,       "0.7.2"}
        , {worker_pool, "4.0.1"}
        , {elvis,       "0.4.3", {pkg, elvis_core}}
+       , {rebar3_format, { git, "https://github.com/alanz/rebar3_format.git"
```

Comment:
If your fixes are reviewed soon we can just wait for that to happen.
If not and we need a fork, I suggest we create forks under the `erlang-ls` organization, so we all have access.

Reply:
I got it working with the unpatched version, so this is not needed.